docs/technical-doc/Endorsements.rst

```diff
@@ -0,0 +1,118 @@
+============
+Endorsements
+============
+
+Intro
+=====
+
+
+Massa uses the Proof-of-Stake selection mechanism with Nakamoto consensus.
+In that context, when there are multiple cliques in close competition,
+we want all nodes to converge towards a single clique as fast as possible to minimize finality time and maximize the quality of the consensus.
+To achieve this, we draw inspiration from Tezos and introduce the concept of Endorsement.
+
+Basic principle
+===============
+
+At every slot `S`, we use the existing Proof-of-Stake selection mechanism to not only draw the block creator for that slot,
+but also `E` other stakers that are responsible for creating Endorsements that can be included in the blocks of slot `S`.
```

Comment:
```suggestion
but also `E` other stakers that are responsible for creating Endorsements that can be included in the block of slot `S`.
```

Reply:
We can find only one block by slot right? :thinking:

Reply:
There can be multiple blocks at a given slot, but they would necessarily be from different cliques.

Example: I am selected to create a block a slot S but instead I create 2 different blocks at that slot.